ZeCamp/ZeCamp/Sources/Schedule/ScheduleScreen.swift

```diff
@@ -34,6 +70,9 @@ struct ScheduleScreen {
         let label = UILabel()
         label.text = "ZeCamp commit \(Bundle.main.shortCommitId!).\n© 2017 Zuhlke Engineering Ltd. ✨"
         label.numberOfLines = 0
+        label.textColor = UIColor.gray
+        label.font = UIFont(name: "Helvetica", size: 15)
```

Comment:
Is there a reason we're setting the font to Helvetica? No reason not to use the system font here.

Reply:
As a Lead UI Designer of this project and Art Director I thought it would be a very good idea.